Compilation failure

There is an error during compilation on Debian sid: files in `src/prog` cannot include `gpm.h`.  Here is the message:

```
Makefile:90: recipe for target 'dep' failed
make[1]: [dep] Error 1 (ignored)
# create dependencies
for DEPS in `echo *.c */*.c`; do \
gcc -I. -I /home/doc/src/gpm/src -M  -I/home/doc/src/gpm/src -DHAVE_CONFIG_H -include headers/config.h -Wall -DSYSCONFDIR="\"/usr/local/etc\"" -DSBINDIR="\"/usr/local/sbin\"" -D_GNU_SOURCE  $DEPS | \
/bin/sed 's/^\(.*\)\.o\([ :]+\)/\1.o \1.lo\2/g' >> .depend ; done
prog/display-buttons.c:40:57: fatal error: gpm.h: No such file or directory
compilation terminated.
prog/display-coords.c:42:57: fatal error: gpm.h: No such file or directory
compilation terminated.
prog/get-versions.c:25:57: fatal error: gpm.h: No such file or directory
compilation terminated.
```

Steps to reproduce it:

```
./autogen.sh
./configure
make
```

--- a/src/Makefile.in
+++ b/src/Makefile.in
@@ -96,7 +96,7 @@ $(DEPFILE) dep: prog/gpm-root.c
 
 	# create dependencies
 	for DEPS in `echo *.c */*.c`; do \
-	$(CC) -I. -I $(srcdir) -M @CPPFLAGS@ $(CPPFLAGS) $$DEPS | \
+	$(CC) -I. -I $(srcdir) -I $(srcdir)/headers -M @CPPFLAGS@ $(CPPFLAGS) $$DEPS | \
 	$(SED) 's/^\(.*\)\.o\([ :]+\)/\1.o \1.lo\2/g' >> $(DEPFILE) ; done
 
 ### INSTALL

